pages/repo/preview.js

```diff
@@ -34,9 +34,15 @@ const PreviewPage = ({ router, data = {} }) => {
   const store = initLocalStore(storeKey)
   let localState = store.get('editorState')
 
-  const schema = getSchema(
-    localState?.meta?.template || document?.meta?.template
-  )
+  const template = localState?.meta?.template || document?.meta?.template
+
+  const schema = useMemo(() => {
+    if (!template) {
+      return
+    }
+    return getSchema(template)
+  }, [template])
+
```

Comment:
```suggestion
  const notFound = !schema ? t('preview/404') : undefined
```
add a message like:
`Dokument oder Schema nicht verfügbar.`

and then at the loader do `error={error || notFound}`

Reply:
done